src/utils/parseJsDoc.js

```diff
@@ -28,7 +29,14 @@ function getType(tag) {
   if (!tag.type) {
     return null;
   }
-  return {name: tag.type.name};
+  return {name: tag.type.name ? tag.type.name : tag.type.expression.name};
+}
+
+function getOptional(tag) {
+  if (tag.type && tag.type.type && tag.type.type === 'OptionalType') {
+    return true;
+  }
+  return;
```

Comment:
Actually I intentionally return undefined (a little hacky) because I didn't want to always add the property and so as not to break existing tests.